test/README.md

```diff
@@ -27,15 +27,6 @@ For `--base <base>`, the default names are `<base>@example.com`, `<base>_ws` and
 
 For `--vcs-provider <vcs>`, supported providers include Gitlab (gitlab), GitHub Enterprise (GHE) and Bitbucker Server (bbs).
 
-**Troubleshooting**
-
-If you get an error from Ruby/Bundler like:
-
-    Could not find tzinfo-1.2.6 in any of the sources
-    Run `bundle install` to install missing gems.
-
-You need to remove the `Gemfile.lock` from the root of the repository.
```

Comment:
This troubleshooting section was specific to the Docker wrapper, so it is no longer relevant.